runtime/laos/src/configs/transaction_payment.rs

```diff
@@ -1,24 +1,95 @@
 use crate::{
-	currency::MICROUNIT,
-	types::{ToAuthor, WeightToFee},
+	currency::{GIGAWEI, KILOWEI},
+	types::ToAuthor,
 	Balance, Balances, Runtime, RuntimeEvent,
 };
-use frame_support::{parameter_types, weights::ConstantMultiplier};
+use frame_support::weights::{
+	ConstantMultiplier, WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial,
+};
 use polkadot_runtime_common::SlowAdjustingFeeUpdate;
+use smallvec::smallvec;
+use sp_core::{ConstU128, ConstU8};
+use sp_runtime::Perbill;
+
+// Provide a common factor between runtimes based on a supply of 10_000_000 tokens.
+pub const SUPPLY_FACTOR: Balance = 100;
```

Comment:
what's the meaning of this constant ?

Moonbeam makes the deposit calculation dependent on it : https://github.com/moonbeam-foundation/moonbeam/blob/b2b9dfeff44dc07ce407b224a98159e33a3e073f/runtime/moonbase/src/lib.rs#L150